wiki/Tournaments/GTS/RGTS_2022/en.md

```diff
@@ -0,0 +1,187 @@
+
+## Mappools
+
+*Every mappool will be announced 7 days (usually Sunday) before the next stage's match timeline.*
+
+## Ruleset
+
+### General rules
+
+1. This tournament is a **2v2 team tournament** with teams of **5 players maximum**.
+2. The teams will first go through Qualifiers, then Group stage, then finally a double-elimination bracket.
+3. The scoring system used will be **Score V2**.
+4. Any player who meets the requirements stated in the [restrictions and registrations](#restrictions-and-registrations) section is allowed to sign up. Note that some players may be declined participation as part of tournament screening.
+5. Any members of staff **apart from organisers, mappool selectors, and referees** are allowed to play.
+6. Players and staff members must have read this ruleset entirely.
+7. All participants must stay respectful and keep a proper attitude. Not following this rule can result in a ban/blacklist from the tournament. This rule concerns all staff members as well.
+8. Any rule changes or unexpected occurrences will be announced in the Discord server.
+
+### Restrictions and registrations
+
+1. In order to participate, players need to be in the rank range of #1,500 - #4,500.
+2. Players are not allowed to rank up further than #1,500 during the tournament. Passing this limit will result in disqualification.
+3. Players must join the Discord server of the tournament to actually be counted as a player.
+4. Players sign up individually. When their registration is accepted, they will get a player role with their respective country.
+5. osu! staff can refuse players from participating in the tournament if their skill is deemed much higher than the targeted rank range.
+6. 2 teams for a country may be allowed under certain conditions:
+   - There are **7 or more players** signed up for a country.
+   - Each team has to have **at least 3 players**.
+7. Teams are otherwise assembled in the following way:
+   - If there are 5 players or less signed up for a country, those players will form the team representing their country.
+   - In case there are more than 10 players signed up for a country, players will decide among themselves who will partake in the tournament. If they cannot decide, the top 10 players by ranking will be chosen to form the 2 teams.
+   - If there are 6 players, players will have to decide on the 5 players who will partake in the tournament. If they cannot decide, the top 5 players by ranking will be chosen to form the team.
+   - If there are 7 to 10 players (after keeping the best ranked ones) signed up for a country, the players from that country will have a few days to decide how to split themselves between the 2 teams. It is recommended to come to an agreement as fast as possible and decide on a captain to aid in scheduling.
+   - In case the players cannot come to an agreement, they will put in 2 teams with at least 3 members in each in order of global ranking.
```

Comment:
```suggestion
   - In case the players cannot come to an agreement, they will be put in 2 teams with at least 3 members in each in order of global ranking.
```